cms/djangoapps/contentstore/views/course.py

```diff
@@ -810,6 +810,15 @@ def _create_or_rerun_course(request):
                     status=400
                 )
 
+        # Allow user to create the course only if they belong to the organisation
+        if settings.FEATURES.get('RESTRICT_NON_ORG_COURSE_CREATION'):
```

Comment:
@farhaanbukhsh, you have to write unit tests for this.

Reply:
@lgp171188 yup, i was waiting for you to verify the approach and then I can write them. :) so is this approach good enough?

Reply:
@farhaanbukhsh, it might also be a good idea to add this sort of a check for library creation.

Reply:
@lgp171188 looks like a good idea.

Reply:
@lgp171188  added the check